.github/ISSUE_TEMPLATE/3-docs-bug.yaml

```
@@ -32,17 +37,22 @@ body:
   - type: textarea
     id: exception-or-error
     attributes:
-      label: If this problem caused and exception or error, please provide it here
+      label: If this problem caused an exception or error, please paste it here
       render: true
+      placeholder: |
+        ```
+        Paste the exception or error here inside a markdown code block,
+        which is annotated by three gave characters (```) before and after the text.
```

```suggestion
        which is annotated by three grave characters \(```\) before and after the text.
```